app/views/analytics/index.html.erb

```diff
@@ -153,11 +153,45 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
+  <!-- Editors List-->
+    <div class="col-sm-4">
       <div class="chart-wrapper">
         <div class="chart-title background-orange">
+          Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+        </div>
+        <div class="chart-stage">
+         <table class='table leader-list'>
+           <tr>
+            <th></th>
+            <th>Editor</th>
+            <th>Updates</th>
+           </tr>
+
+           <% @updates.group(:whodunnit).order('count_id DESC').limit(10).count(:id).each_with_index do |who, i| %>
```

Comment:
Yeah, not a fan of putting actual queries in views.  In fact, there are several issues about moving these types of code snippets out of the views and somewhere else, like controllers and helpers.  Please do that here.

Reply:
I can move it.  I may have to do some research on controller vs helper methods...